docs/artifactory/replicationjobdelete/help.go

```diff
@@ -0,0 +1,8 @@
+package replicationdelete
+
+const Description = "Remove a replication repository job from Artifactory."
+
+var Usage = []string{`jfrog rt rjd <repository key>`}
+
+const Arguments string = `	repository key
+		The repository for which the replication configuration will be delete.`
```

Comment:
The repository for which the replication configuration will be delete.` -->
The repository from which the replication configuration will be deleted.`